src/annotator/anchoring/test/fake-pdf-viewer-application.js

```diff
@@ -19,6 +19,8 @@ import { RenderingStates } from '../pdf';
  * @typedef PDFJSConfig
  * @prop {boolean} newTextRendering - Whether to emulate the PDF.js text rendering
  *   changes added in v2.9.359.
+ * @prop {[number, number, number, number]} pageBoundingBox - The [x0, y0,
+ *   x1, y1] coordinates of the crop box for PDF pages.
```

Comment:
The changes here allow tests to customize the crop box for PDF pages and make sure that the various APIs that transform between PDF and viewport coordinates are in sync.